README.md

```diff
@@ -783,73 +785,68 @@ Here are a few ways you might hear of or see in documents:
 
 ### Classes of stock
 
-🄳 Often investors ask for rights to be paid back first in exchange for their investment.
+🄳 Investors often ask for rights to be paid back first in exchange for their investment.
 The way these different rights are handled is by creating different
-[**classes of stock**](https://www.upcounsel.com/classes-of-stock), each with its own.
-These are also sometimes called
-[**classes of shares**](https://www.investopedia.com/terms/c/class.asp) (though that term has
-another meaning in the context of mutual funds).
+[**classes of stock**](https://www.upcounsel.com/classes-of-stock). (These are also sometimes
+called [**classes of shares**](https://www.investopedia.com/terms/c/class.asp), though that term
+has another meaning in the context of mutual funds.)
 
 🄳 Two important classes of stock are
 [**common stock**](https://en.wikipedia.org/wiki/Common_stock) and
 [**preferred stock**](https://en.wikipedia.org/wiki/Preferred_stock). In general, preferred stock
-is stock that has rights, preferences, and privileges that common stock does not have.
+has “[rights, preferences, and privileges](https://www.americanbar.org/publications/blt/2014/01/04_bigler.html)"
```

Comment:
non-oriented 2nd quote (note that once things are un-balanced like this Flowmark won't fix either)